mne/cov.py

```diff
@@ -1125,10 +1128,11 @@ def prepare_noise_cov(noise_cov, info, ch_names, rank=None,
     else:
         C = np.diag(noise_cov.data[C_ch_idx])
 
-    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
+    _scalings = DEFAULTS['scalings']
```

Comment:
the values here are quite different, but the DOC indicates that the DEFAULTS will be used.